app/src/components/Translations.vue

```diff
@@ -0,0 +1,53 @@
+<script>
+  var Vue = require('vue')
+  var VueI18n = require('vue-i18n')
+
+  // ready translated locales
+  var locales = {
+    de: {
```

Comment:
Können wir die einzelnen Language-Files nicht auslangern in Sprachdateien?
Idealerweise hat jede Komponente für jede Sprache eine eigene JSON.

z.B. `Kontakt_de.json`, `Kontakt_en.json`

Das würde die Translations (vielleicht?) sehr viel übersichtlicher machen.